info/README.md

```diff
@@ -0,0 +1,7 @@
+UPDATE MAIN README
+
+* To update the codestyle text in the main README, you **NEED TO CHANGE** the content of the file `guide-chapter-<N>.md`, contained in `info/guide`, the corresponding section of the rules that you changed / added.
+
+* After the updates are made **NEED TO PERFORM GENERATION** using gradle `generateFullDoc` and `generateAvailableRules` contained in `info/build.gradle.kts` file. Example - `./gradlew :generateFullDoc` and `./gradlew :generateAvailableRules`.
```

Comment:
After the up[dates are made, you **NEED**...

Reply:
Also, DRY (don't repeat yourself). I's sufficient to provide a code snippet (use the `console` language):

> ... navigate to the `info/` sub-directory and re-generate the documentation using _Gradle_:

```console
$ cd info/
$ ./gradlew :generateFullDoc
$ ./gradlew :generateAvailableRules
```

Reply:
Done [78342c5](https://github.com/saveourtool/diktat/pull/1384/commits/78342c53dd92a42802f781323a6cc10b03605118)

Please check it again